packages/core/lib/node/to-readable.ts

```diff
@@ -0,0 +1,47 @@
+import { Readable } from "stream";
```

Comment:
There is no need for this module, [`Readable.from()`](https://nodejs.org/dist/latest-v16.x/docs/api/stream.html#streamreadablefromiterable-options) does it all in a fast way.

Reply:
Thank you for this input, I changed it within https://github.com/contra/graphql-helix/pull/228/commits/9ab1cd321900e47022e516c2b210ab9e01f026b6